demand_land.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: index
+title: Demand for food, forestry, etc.
+prev: inputs_demand.html
+next: outputs_quantity.html
+gcam-version: v5.3 
+---
+
+# Table of Contents
+
+- [Inputs to the Model](#inputs-to-the-model)
+- [Description](#description)
+- [Equations](#equations)
+- [Insights and intuition](#insights-and-intuition)
+- [Policy options](#policy-options)
+- [IAMC Reference Card](#iamc-reference-card)
+- [References](#references)
+
+## Inputs to the Model
+**Table 1: Inputs required by the demand model <sup>[1](#table_footnote)</sup>**
+
+| Name | Resolution | Unit | Source |
+| :--- | :--- | :--- | :--- |
+|  |  |  | [Exogenous](inputs_demand.html) |
+
+
+## Description
+
+{Add as many subsections and sub-subsections as needed to describe this area. Each subsection should have a short paragraph qualitatively describing the topic. Additional information can be included on a details_xyz.md page if needed}
+
+
+## Equations 
+The equations that determine food, feed, and forest demand are described here.
+
+#### Variable #1
+
+{Insert equations used to calculate variable #1}
+
+See `method name` in [code_file.cpp](link to code on GitHub).
+
+## Policy options 
+This section summarizes some of the energy-based policy options available in GCAM. 
+
+### Policy type #1
+
+{Insert paragraph describing a type of policy for this sector. Include links to xml input files on GitHub and/or the policies.html or policies_examples.html pages}
+
+## Insights and intuition
+
+### Paper/Topic #1
+
+{One paragraph summary of a key insight from one or more papers}
+
+## IAMC Reference Card
+
+Agriculture and forestry demands
+- [X] Agriculture food
+- [X] Agriculture food crops
+- [X] Agriculture food livestock
+- [X] Agriculture feed
+- [X] Agriculture feed crops
+- [X] Agriculture feed livestock
+- [X] Agriculture non-food
+- [X] Agriculture non-food crops
+- [X] Agriculture non-food livestock
+- [X] Agriculture bioenergy
+- [X] Agriculture residues
+- [X] Forest industrial roundwood
+- [ ] Forest fuelwood
+- [X] Forest residues
+
```

Comment:
This page apparently needs a lot more work. 
For IAMC reference card, our forest data included forest fuelwood. It is not disaggregated or linked to energy demand, though this page is for Ag demand.